cli/tests/unit/test_util.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
 import { assert, assertEquals } from "../../../std/testing/asserts.ts";
+import * as colors from "../../../std/fmt/colors.ts";
+export { colors };
```

Comment:
Seems like unrelated changes?

Reply:
It's re-exported for `unit_test_runner.ts` (the `export * as ...` syntax actually isn't standard yet).